### PR TITLE
Game result screen

### PR DESCRIPTION
**Pull request by Epix37**
Based on choices-blur PR.

This adds somewhat of a game result screen at the end of the game. 
victory

- Both players blurred, defeated player desaturated.
- I couldn't find a good backdrop for the text. Game assets don't really fit in with the joust style here.
- Text is currently centered across the whole screen. Not sure if 75% (board) would be better.

### CODE REVIEW

File: less/joust.less [hunk 1]
```diff
@@ -74,6 +74,15 @@
 	}
 }
 
+.player.inactive-colored {
+	.short, .tall {
+		-webkit-filter: blur(3px);
+		-moz-filter: blur(3px);
+		-o-filter: blur(3px);
+		filter: blur(3px);
```
Comment (beheh) [inline less/joust.less]: Remove the vendor prefixes here.

**Comment by beheh** [comment]
We can probably change the text to "{Playername} wins!" or similar at some point, since the playername will also be blurred out. This is fine as is now though, thanks.

**Comment by Epix37** [comment]
epixwins


Hm... I actually think while we just have pure text "Victory!" looks better. But I agree integrating the name in some fashion is a good idea.

**Comment by beheh** [comment]
Landed in 8df42d1edae5c965fea3ce316a4449e68ef11344.

**Pull request closed without merging**